docs/user-guide/local.md

```diff
@@ -0,0 +1,207 @@
+export:
+hello: $ echo $GREETING
+hello: Hello, world!
+hello:
+set-metadata: $ meta set example.coverage 99.95
+set-metadata:
+```
+
+
+The build artifacts are created under the "./sd-artifacts" directory after the build is finished.
+ 
+```bash
+$ ls ./sd-artifacts
+builds.log       environment.json steps.json
+```
+
+
+# config command
+You can configure sd-local settings in key/value format by config command.
+Your settings are stored in `~/.sdlocal/config`.
+
+
+## Usage
+
+
+### set subcommand
+You can configure sd-local in key/value format. You have to set `api-url`, `store-url` and `token` to execute builds.
+Please refer to [List of the keys](#list-of-the-key) about available settings.
+```bash
+$ sd-local config set <key> <value>
+```
+
+
+### view subcommand
+You can confirm the current settings.
+
+
+```bash
+$ sd-local config view
+```
+
+
+### List of the key.
+
+|key|description|
+|---|---|
+|api-url|Please specify the URL of Screwdriver API of the cluster you are using.|
+|store-url|Please specify the URL of Screwdriver Store of the cluster you are using.|
+|token|Please specify the [API token](api#authentication-and-authorization)|
+|launcher-image|Please specify the launcher image. (default: `screwdrivercd/launcher`)|
+|launcher-version|Please specify the launcher version. (default: `stable`)|
+
+
+# build command
+This command run the build in your local environment.
+
+
+## Usage
+Run build with specific the job. Even if a job is succeeded, any other jobs are not triggered.
+```bash
+$ sd-local build <job name>
+```
+
+
+It starts the job based on `screwdriver.yaml` in the current directory, and the build artifacts are created under the "./sd-artifacts" directory.
+
+
+## Options
+The following options can be used in build command.
+
+
+|option|describe|
+|---|---|
+|--artifacts-dir|Specify the destination of build artifacts. (default: `./sd-artifacts`)|
+|-e, --env|Set the environment variables in the build environment in `<key>=<value>` formats. (Multiple specifications allowed)|
+|--env-file|Set the environment variables in the build environment in file format. (Refer to [env-file format](#env-file-format))|
+|-m, --memory|Specify the limit value of the memory resource of the build environment. (Can be specified in b, k, m, g memory unit)|
+|--meta|Specify [metadata](metadata) used in build environment. (JSON format)|
+|--meta-file|Specify [metadata](metadata) used in build environment in the file. (JSON format)|
+|--src-url|Specify the remote SCM URL to build. (`https` or `ssh` format)|
+
+
+Note:
+- sd-local uses the value specified by `--env` option, if a same environment variable is specified in both `--env` and `--env-file` options.
+- You can't use both `--meta` and `--meta-file` options.
```

Comment:
```suggestion
|option|description|
|---|---|
|--artifacts-dir|The build artifacts destination (default: `./sd-artifacts`)|
|-e, --env|The environment variables in a build environment using `<key>=<value>` format (multiple variables allowed)|
|--env-file|The environment variables in a build environment using file format (refer to [env-file format](#env-file-format))|
|-m, --memory|The memory limit of the build environment (can be specified in b, k, m, or g memory unit)|
|--meta|The [metadata](metadata) used in a build environment using string JSON format|
|--meta-file|The [metadata](metadata) used in build environment using file format|
|--src-url|The remote SCM URL to build (`https` or `ssh` format)|


Note:
- sd-local uses the value specified by `--env` option, if a same environment variable is specified in both `--env` and `--env-file` options.
- You can't use both `--meta` and `--meta-file` options.
```

Reply:
Might be good to have some examples for these, but not necessary

Reply:
I fixed.
Added example of "--meta"  option.